phoenix-spark-transactions/README.md

```diff
@@ -0,0 +1,152 @@
+# Prerequisites
+
+1. You have set your CDP workload password:
+
+https://docs.cloudera.com/management-console/cloud/user-management/topics/mc-setting-the-ipa-password.html
+
+2. You have synchronized users from User Management Service in the CDP Control Plane into the environment
+in which your COD database is running.
+
+3. Ensure CDE service enabled and virtual cluster created in Data Engineering Experiance
+
+https://docs.cloudera.com/data-engineering/cloud/enable-data-engineering/topics/cde-enable-data-engineering.html
+https://docs.cloudera.com/data-engineering/cloud/manage-clusters/topics/cde-create-cluster.html
+
+# Set HBase and Phoenix verions in your Maven project
+
+From the `describe-client-connectivity` call, we can get the HBase and Phoenix version information. This code snippet
+shows fetching the database connectivity information and parsing the required HBase and Phoenix information to build your
+application.
+```
+echo "HBase version"
+cdp opdb describe-client-connectivity --database-name my-database --environment-name my-env | jq ".connectors[] | select(.name == \"hbase\") | .version"
+echo "Phoenix Connector Version"
+cdp opdb describe-client-connectivity --database-name my-database --environment-name my-env | jq ".connectors[] | select(.name == \"phoenix-thin-jdbc\") | .version"
+```
+
+
+```
+HBase Version
+2.4.6.7.2.14.0-133
+Phoenix Spark Version
+"6.0.0.7.2.14.0-133"
+```
+
+Finally, update the HBase and Phoenix Connector versions in our Maven project/configuration.
+```
+<properties>
+...
+    <phoenix.connector.version>6.0.0.7.2.14.0-133</phoenix.connector.version>
```

Comment:
You don't need the phoenix connector version

Reply:
Removed it.